s3/pkg/service/bucket.go

```diff
@@ -237,11 +242,9 @@ func (s *s3Service) GetBucketLifecycle(ctx context.Context, in *pb.BaseRequest,
 		return nil
 	}
 
-	_, _, err = CheckRights(ctx, bucket.TenantId)
+	_, _, _, err = CheckRights(ctx, bucket.TenantId)
 	if err != nil {
```

Comment:
Here need to return ErrBucketAccessForbidden,  otherwise the user may create a new one with the same name, but it will failed because duplicate key.